source/developer/developer-setup.md

```diff
@@ -90,6 +90,14 @@ Any issues? Please let us know on our forums at: http://forum.mattermost.org
 
 ### Archlinux ###
 
+#### Build and install with AUR ####
```

Comment:
@gabx, when we use the term "unofficial" in our documentation it refers to the fact that Mattermost Inc. does not maintain or provide support for the package.

I've submitted a PR to your branch with the requested changes.